.gitignore

```diff
@@ -0,0 +1,20 @@
+# Ignore Visual Studio Code settings
+.vscode/
+
+# Ignore executable files
+*.exe
+
+# Ignore object files
+*.obj
+
+# Ignore program database files
+*.pdb
+
+# Ignore incremental linking files
+*.ilk
+
+# Ignore temporary files
+*.tmp
+
+# Ignore .gitignore file
+*..gitignore
```

Comment:
should not be ignoring `.gitignore` file.

Reply:
issue resolved in
https://github.com/Muhammadali2025222/C-Practice/pull/5/files